runtime/runtime/src/config.rs

```diff
@@ -73,6 +70,41 @@ macro_rules! safe_add_balance_apply {
     }
 }
 
+pub fn transfer_exec_fee(
```

Comment:
I am confused. You implement this here and again in `VMLogic`?

Reply:
Yes. I would unite these functions, but the version in `VMLogic` is supposed to throw `HostError::IntegerOverflow` in case of overflows, and version in `Runtime` doesn't.
Is there actually any confidence that addition of such fees will never cause overflow? If so, I could just reuse this function.

Reply:
Discussed offline, code duplication was removed